tst/grade.test.ts

```diff
@@ -0,0 +1,129 @@
+import { server } from './server';
+import mongoose from 'mongoose';
+import supertest from 'supertest';
+import Applicant from '../src/models/applicant';
+import ScreeningGrade from '../src/models/screening-grade';
+import InterviewGrade from '../src/models/interview-grade';
+
+describe("Applicant's ScreeningGrade + InterviewGrade Get", () => {
+    const ENDPOINT = process.env.MONGO_URL;
+
+    beforeEach((done) => {
+        mongoose.connect(ENDPOINT, { useNewUrlParser: true, useCreateIndex: true, useUnifiedTopology: true });
+        mongoose.connection.on('connected', () => done());
+    });
+
+    afterEach((done) => {
+        mongoose.connection.db.dropDatabase(() => {
+            mongoose.connection.close(() => done());
+        });
+    });
+
+    test('GET /applicant', async () => {
+        const applicant = await Applicant.create({
+            firstName: 'John',
+            lastName: 'Doe',
+            email: 'johndoe@gmail.com',
+            role: 'Developer',
+            major: 'Combined CS + Stat',
+            yearStanding: 1,
+            level: 'Beginner',
+            status: 'Pending',
+        });
+
+        await supertest(server)
+            .get('/applicant')
+            .expect(201)
+            .then((response) => {
+                // Check type and length
+                expect(Array.isArray(response.body)).toBeTruthy();
+                expect(response.body.length).toEqual(1);
+
+                // Check data
+                expect(response.body[0]._id).toBe(applicant._id.toString());
+                expect(response.body[0].firstName).toBe(applicant.firstName);
+                expect(response.body[0].lastName).toBe(applicant.lastName);
+            });
+    });
```

Comment:
Can we restructure the folders a little bit? Can you create an /api/ folder under tst and put the applicant tests under /api/applicant.test.ts and grade tests under /api/grade.test.ts?